cmd/outline-ss-server/config_example.yml

```
@@ -1,15 +1,23 @@
-keys:
-  - id: user-0
-    port: 9000
-    cipher: chacha20-ietf-poly1305
-    secret: Secret0
+services:
+  - listeners:
+      - type: direct
+        address: "tcp://[::]:9000"
+      - type: direct
+        address: "udp://[::]:9000"
```

To make it more user friendly, I wonder if we should accept a string as "direct"

```suggestion
      - "tcp://[::]:9000"
      - "udp://[::]:9000"
```

Go Task does that.

---

I meant we could have both. Long form is the canonical, with the short form as convenience. Caddy does that.